apps/meteor/server/startup/migrations/v299.ts

```diff
@@ -0,0 +1,27 @@
+import { Settings } from '@rocket.chat/models';
+
+import { settingsRegistry } from '../../../app/settings/server';
+import { addMigration } from '../../lib/migrations';
+
+addMigration({
+	version: 299,
+	async up() {
+		const customOauthServices = await Settings.find({ _id: /Accounts_OAuth_Custom-[^-]+$/im }).toArray();
```

Comment:
the previous one was working, but since there are other change requests, I thought it was good to add this one as well
```suggestion
		const customOauthServices = await Settings.find({ _id: /Accounts_OAuth_Custom-[^-]+$/ }, { projection: { _id: 1 } }).toArray();
```